src/NServiceBus.Metrics.ServiceControl/ServiceControlReporting/NServiceBusMetricReport.cs

```diff
@@ -27,8 +28,13 @@ public void RunReport()
                 dispatcher.Send(new TransportMessage(Guid.NewGuid().ToString(), headers)
                 {
                     Body = body,
-                    MessageIntent = MessageIntentEnum.Send
-                }, new SendOptions(destination));
+                    MessageIntent = MessageIntentEnum.Send,
+                    TimeToBeReceived = ttbr,
+                }, new SendOptions(destination)
+                {
+                    EnlistInReceiveTransaction = false,
+                    TimeToBeReceived = ttbr
```

Comment:
If I look at the heartbeats code then TTBR is set only once and that is on the `TransportMessage`:

https://github.com/Particular/ServiceControl.Plugin.Nsb5.Heartbeat/blob/master/src/ServiceControl.Plugin.Nsb5.Heartbeat/ServiceControlBackend.cs#L28-L31

Reply:
Ok. will remove from `SendOptions`